### PR TITLE
Command misc mappings

### PR DESCRIPTION
**Pull request by maxpowa**
Bunch of mappings related to `ICommand` and `CommandBase` implementations.

### CODE REVIEW

File: mappings/net/minecraft/block/tile/impl/TileEntitySign.mapping [hunk 1]
```diff
@@ -11,8 +17,14 @@ CLASS none/art net/minecraft/block/tile/impl/TileEntitySign
 		METHOD i_ getNameComponent ()Lnone/fa;
 		METHOD y_ shouldEchoCommandFeedback ()Z
 	CLASS none/art$2
+		METHOD a hasPermission (ILjava/lang/String;)Z
+			ARG 0 permissionLevel
+			ARG 1 commandName
 		METHOD a appendCommandFeedback (Lnone/fa;)V
 			ARG 0 component
+		METHOD a updateCommandStat (Lnone/o$a;I)V
```
Comment (asiekierka) [inline mappings/net/minecraft/block/tile/impl/TileEntitySign.mapping]: What is a CommandStat? I'm genuinely curious.

File: mappings/net/minecraft/client/player/EntityPlayerClientAbstract.mapping [hunk 2]
```diff
@@ -1 +1,3 @@
 CLASS none/bob net/minecraft/client/player/EntityPlayerClientAbstract
+	METHOD y isSpectator ()Z
```
Comment (asiekierka) [inline mappings/net/minecraft/client/player/EntityPlayerClientAbstract.mapping]: I was never sure about these. isSpectator/isCreative vs. isModeSpectator/isModeCreative?

**Comment by asiekierka** [comment]
LGTM beyond that one comment

**Comment by maxpowa** [comment]
Removed the two mappings that were slightly ambiguous, they weren't really related directly to the rest of the mappings here anyways.

**Pull request merged**